src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Component.razor

```diff
@@ -3,6 +3,35 @@
 
 <hr />
 
+<div class="spin-button-components-container">
+    <h2>BitSpinButton </h2>
+
+    <h4># Basic SpinButton</h4>
+    <BitSpinButton Min="0" Max="100" Step="1" Label="Basic SpinButton"></BitSpinButton>
+    <BitSpinButton Min="-10" Max="10" Step="0.1" Label="Decimal SpinButton"></BitSpinButton>
+    <BitSpinButton DefaultValue="25" Min="0" Max="100" Step="1" Label="Disabled SpinButton" IsEnabled="false"></BitSpinButton>
+    <h4># SpinButton with icon</h4>
+    <BitSpinButton IconName="IncreaseIndentLegacy" Min="0" Max="100" Step="1" Label="With icon"></BitSpinButton>
+    <BitSpinButton IconName="IncreaseIndentLegacy" Min="0" Max="100" Step="1" Label="Disabled with icon" IsEnabled="false"></BitSpinButton>
+    <h4># SpinButton with suffic</h4>
+    <BitSpinButton Suffix="Inch" Min="0" Max="100" Step="1" Label="With suffix"></BitSpinButton>
+
+    <h4># SpinButton with custom handler</h4>
+    <BitSpinButton DefaultValue="5" Min="0" Max="100" Step="1" Label="With custom handler" OnChange="@(e => HandleOnChange(e))"></BitSpinButton>
+    <p>@Message</p>
+
+    <h4># SpinButton with label above</h4>
+        <BitSpinButton DefaultValue="7" Suffix="cm" Min="0" Max="100" Step="1" Label="With labal above" LabelPosition="@LabelPosition.top"></BitSpinButton>
+</div>
+
+@code{
```

Comment:
please move this "@code" section into the code behind file (Component.razor.cs)

Reply:
done